openprescribing/measure_definitions/lyrica.json

```diff
@@ -29,14 +29,34 @@
   "low_is_good": true,
   "numerator_type": "bnf_quantity",
   "numerator_bnf_codes_query": [
-    "SELECT bnf_code FROM {hscic}.presentation WHERE name LIKE '%_Cap%'"
```

Comment:
We're now including tabs where the previous query included only caps. Just wanted to confirm that this was intentional.

Reply:
Yes, thanks.